RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -288,14 +292,20 @@
 
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(pixelLessStep, pixelLessStepClassifier1.clone(
+	mva = dict(GBRForestLabel = 'MVASelectorPixelLessStep_Phase1'),
+	src = 'pixelLessStepTracks',
+	qualityCuts = [-0.4,0.0,0.4]
+))
+(trackingPhase1 & fastSim).toModify(pixelLessStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this is not needed and may be a bit confusing re pp_on_AA  below: the fastsim value is inherited from L281 and this line can be removed

Reply:
Removed unnecessary line